lib/ansible/plugins/inventory/gcp_compute.py

```diff
@@ -232,6 +232,8 @@ def hostname(self):
                 name = self._get_privateip()
             elif order == "name":
                 name = self.json[u"name"]
+            elif order == "hostname":
+                name = self.json[u"hostname"] if u"hostname" in self.json else self.json[u"name"]
```

Comment:
Can you change this to a normally formatted if/else block?

Reply:
Done.  I also converted the parent if/then to a switch case.